src/OrbitGl/GraphTrack.cpp

```diff
@@ -136,7 +136,7 @@ float GraphTrack<Dimension>::GetLabelYFromValues(
 template <size_t Dimension>
 std::string GraphTrack<Dimension>::GetLabelTextFromValues(
     const std::array<double, Dimension>& values) const {
-  const std::array<std::string, Dimension>& series_names = series_.GetSeriesNames();
+  const std::vector<std::string>& series_names = series_.GetSeriesNames();
   std::optional<uint8_t> value_decimal_digits = series_.GetValueDecimalDigits();
   std::string value_unit = series_.GetValueUnit();
   std::string text;
```

Comment:
I can add this comment to the proper line, that's why it's here:

Below there are two calls to `series_names[i]`. Can you replace that by `series_names.at(i)` so that it is checked whether the i-th element is still part of the vector.